tmt/convert.py

```diff
@@ -272,19 +272,31 @@ def read_datafile(path, filename, datafile, types, testinfo=None):
             echo(style('environment:', fg='green'))
             echo(pprint.pformat(data['environment']))
 
+    def checked_name(name):
```

Comment:
just a minor thing, but why did you called this function `checked_name()`?

Reply:
Because the return value is 'name' (of rpm) which is 'checked' ;)
If you have better name....

Reply:
maybe `negated_` I don't know. Checked doesn't really fit imo. But yeah, it's just a minor thing.

Reply:
`sanitized_name`? `normalized_name`? "sanitized" sounds good to me, it's what the function does.

Reply:
Sanitized sounds good, adjusted.